src/core/sys/dragonflybsd/sys/_cpuset.d

```diff
@@ -0,0 +1,30 @@
+/**
+ * D header file for DragonFlyBSD.
+ *
+ * Authors: Martin Nowak, Diederik de Groot(port:DragonFlyBSD)
+ * Copied:  From core/sys/freebsd/sys
+ */
+module core.sys.dragonflybsd.sys._cpuset;
+
+version (DragonFlyBSD):
+extern (C) pure nothrow @nogc @system:
```

Comment:
@joakim-noah Thanks for reviewing this !
They were already defined (e.g freebsd implementation), except for the requested addition of @system.|
Does it have a negative effect to set these attributes ? If could prevent future issues when a function or alias is added.

Reply:
Our previous modules use these attributes inconsistently, since some of them predate the attributes.  Would rather get this right for this new platform from the start, attributes do affect the mangling of D functions, ie the ABI.

Reply:
Ok will work on removing them. Thanks for the explanation !